spec/models/user_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  # tests go here
+  subject { User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.') }
+
+  before { subject.save }
+
+  it 'Should be present' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+
+  # it 'Photo should be present' do
+  #   subject.photo = nil
+  #   expect(subject.photo).to_not be_valid
+  # end
+
+  # it 'Bio should be present' do
+  #   subject.bio = nil
+  #   expect(subject.bio).to_not be_valid
+  # end
```

Comment:
- [ ] I strongly recommend you to remove unused comments from your code. Kindly remove them if they are not necessary :)